terraform/dynamodb.tf

```diff
@@ -45,3 +45,16 @@ resource "aws_dynamodb_table" "sre_bot_data" {
     type = "S"
   }
 }
+
+
+resource "aws_dynamodb_table" "incidents_table" {
+  name           = "incidents"
+  hash_key       = "id"
+  read_capacity  = 1
```

Comment:
The only thing is that those RCU and WCU (read and write capacity) might be a bit low but we can always increase them later on. This means that it does 1 read/write per second.

Reply:
I'll bump it a bit, didn't think we'd be making many calls to the DB but could be worth avoiding issues early on